### PR TITLE
Bootstrap classes

### PR DESCRIPTION
**Pull request by Woulei**
Issue #15

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,14 +1,20 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>TocBears</title>
+    <title>Taste of Code - Free beginner course to learn about programming</title>
```
Comment (arnoFleming) [inline app/views/layouts/application.html.erb]: nice

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
     <%= csrf_meta_tags %>
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
   <body>
+    <nav class="navbar navbar-default navbar-static-top">
+    <%#= render 'shared/navbar' %>
```
Comment (arnoFleming) [inline app/views/layouts/application.html.erb]: please remove unused code

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -1,14 +1,20 @@
...
+    </nav>
     <%= yield %>
+    <footer class="footer">
+      <%#= render 'shared/footer' %>
```
Comment (arnoFleming) [inline app/views/layouts/application.html.erb]: idem

**Comment by arnoFleming** [comment]
Since you're adding a layout, maybe you also want to add it to the controllers that need this layout.

**Comment by Woulei** [comment]
Updated. @arnoFleming please review again

**Pull request merged**